# This msg file specifies the object pose which includes 
# (1) the dimension of the object (x,y,z for box)
# (2) position of the object (x,y,z)
# (3) orientation of the object (quat:x,y,z,w)

float32[] dims
float32[] position
float32[] orientation